### PR TITLE
new dev-python/matrix2latex ebuild

### PR DESCRIPTION
**Pull request by TheChymera**
Replacing #506

### CODE REVIEW

**Comment by TheChymera** [comment]
Ok, I don't understand where the issue with the checks is. Is it :

1) !!! .travis.yml outdated! Update available https://github.com/mrueg/repoman-travis 
or
2) dependency.bad [fatal]        50

In either case, what do I do?

**Comment by marbre** [comment]
Probably fixed with commit 4a8ff1169b .

**Comment by TheChymera** [comment]
@marbre  how can I make the cheks run again?

**Comment by marbre** [comment]
Push your branch again.

File: dev-python/matrix2latex/matrix2latex-1.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+HOMEPAGE="https://github.com/TheChymera/matrix2latex"
+SRC_URI="https://github.com/TheChymera/${PN}/blob/master/archive/${PN}Python-${PV}.tar.gz?raw=true -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+RDEPEND=""
+DEPEND=""
```
Comment (jlec) [inline dev-python/matrix2latex/matrix2latex-1.0.0.ebuild]: Drop empty definitions.

File: dev-python/matrix2latex/matrix2latex-1.0.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python{2_7,3_3,3_4} )
```
Comment (jlec) [inline dev-python/matrix2latex/matrix2latex-1.0.0.ebuild]: Don't add 3.3 to new ebuilds

File: dev-python/matrix2latex/matrix2latex-9999.ebuild [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python{2_7,3_3,3_4} )
```
Comment (jlec) [inline dev-python/matrix2latex/matrix2latex-9999.ebuild]: see above.

File: dev-python/matrix2latex/matrix2latex-9999.ebuild [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+HOMEPAGE="https://github.com/TheChymera/matrix2latex"
+EGIT_REPO_URI="https://github.com/TheChymera/matrix2latex"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS=""
+IUSE=""
+
+RDEPEND=""
+DEPEND=""
```
Comment (jlec) [inline dev-python/matrix2latex/matrix2latex-9999.ebuild]: here as well.

File: dev-python/matrix2latex/matrix2latex-1.0.0.ebuild (hunk 1 above)
Comment (TheChymera) [inline dev-python/matrix2latex/matrix2latex-1.0.0.ebuild]: just for the depends? I see a number of other ebilds have empty IUSE

Comment (jlec) [inline dev-python/matrix2latex/matrix2latex-1.0.0.ebuild]: *DEPEND should be dropped. Although empty IUSE could in principle also be dropped, I like to have it set empty.

**Comment by TheChymera** [comment]
@jlec  @marbre  bump?

**Pull request merged**